.travis.yml

```
@@ -1,15 +1,11 @@
 language: node_js
 node_js:
-    - 6
-    - 7
-    - 8
-    - 9
     - 10
```

Do you think it's worth testing specifically on our minimum supported version similar to the [GitHub Actions tests in eslint/eslint](https://github.com/eslint/eslint/blob/1ee6b6388305a8671c8d4c3cf30c2dbf18a1ff7e/.github/workflows/ci.yml#L29)?

```suggestion
    - "10.12.0"
    - 10
```